### PR TITLE
Allowing updating of confluence QA release space 

### PR DESCRIPTION
**Pull request by timofeee**
This update will add the ability to automatically write a confluence page for each release which is something Carmelo had been doing manually each time.

### CODE REVIEW

File: src/writeConfluence.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+const { markdown } = require('markdown')
+const Confluence = require('confluence-api')
+
+const config = {
+    username: process.env.CONFLUENCE_USER,
+    password: process.env.CONFLUENCE_API_KEY,
+    baseUrl:  "https://ecobee.atlassian.net/wiki"
```
Comment (nataliegirard) [inline src/writeConfluence.js]: Please remove all ecobee specific URLs. It's a public repo

Comment (timofeee) [reply src/writeConfluence.js]: Will do. Forgot this was public.

File: src/writeConfluence.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    const includeReleaseName = name ? `<p><strong>Release Name: ${name}<br /></strong></p>` : ''
+    const html = `${includeTeamName} ${includeReleaseName} <p><strong>Notes:<br /></strong></p> ${markdown.toHTML(body)}`
+
+    const today = new Date();
+    const month = `0${today.getMonth()+1}`.slice(-2)
+    const date = `${today.getFullYear()}-${month}-${today.getDate()}`
+    const title = `${date} - ${repositoryName} ${version}`
+    
+    // Find the Releases page for the current year
+    confluence.getContentByPageTitle(CONFLUENCE_QA_SPACE, `${today.getFullYear()} Releases`, (err, response) => {
```
Comment (diegolamanno) [inline src/writeConfluence.js]: We can probably create a util function to get the parent page title. So it is easier to read and avoid the repetition of
```
today.getFullYear() Releases
```

**Comment by cvolpe-ecobee** [comment]
@timofeee should we be adding a section for 'confluenceSettings' to releaseBuddy.config.json?

**Comment by timofeee** [comment]
Updated to keep things more simple and open where the user will just defined in the config the confluence space and parent page. 

Remove all the things specific to us.

**Comment by cvolpe-ecobee** [comment]
@timofeee, un-related to your work but a good time to ask about this. Shouldn't releaseBuddy.config.json be more of an example file and not include our actual settings since this is a public repo?

**Comment by timofeee** [comment]
> @timofeee, un-related to your work but a good time to ask about this. Shouldn't releaseBuddy.config.json be more of an example file and not include our actual settings since this is a public repo?

There is an example in the README. 

But the tool is configured to announce its own release right now. Not sure where does that slack webhook point to but someone can definitely take that and spam them.

**Comment by cvolpe-ecobee** [comment]
> > @timofeee, un-related to your work but a good time to ask about this. Shouldn't releaseBuddy.config.json be more of an example file and not include our actual settings since this is a public repo?
> 
> There is an example in the README.
> 
> But the tool is configured to announce its own release right now. Not sure where does that slack webhook point to but someone can definitely take that and spam them.

I just checked to make sure that it is not pointing to ours, so it should be ok.

**Comment by cvolpe-ecobee** [comment]
Tested this locally and got a page to post to confluence 
https://ecobee.atlassian.net/wiki/spaces/CW/pages/943391396/2019-08-9+-+Test-repo+v0.1.0

**Pull request merged**